src/main/flight/mixer.c

```diff
@@ -511,6 +513,13 @@ static float calcEzLandLimit(float maxDeflection)
         ezLandLimit = maxDeflection / mixerRuntime.ezLandingThreshold; // normalised 0 - 1
         ezLandLimit = fmaxf(ezLandLimit, mixerRuntime.ezLandingLimit); // stay above the minimum
     }
+    // calculate limit to where the mixer can raise the throttle based on RPY stick deflection
+    // TODO sanity checks like number of sats, dop, accuracy?
+    if (speed < mixerRuntime.ezLandingSpeed && mixerRuntime.ezLandingSpeed > 0.0f) {
+        const float speed_limit = speed / mixerRuntime.ezLandingSpeed;  // normalised 0 - 1
+        DEBUG_SET(DEBUG_EZLANDING, 5, lrintf(speed_limit * 10000.0f));
+        ezLandLimit = fmaxf(ezLandLimit, speed_limit);
```

Comment:
maybe 0-100 would be sufficient as a scale for activity, and ease of interpretation in the log?

Reply:
0-100 would be enough, but all the other ezlanding debug fields are scaled by 10000, so I used that here too to keep things consistent.

Reply:
I made a PR that should scale the EZLANDING debug fields properly and display them as percentages.
https://github.com/betaflight/blackbox-log-viewer/pull/712